hawc/apps/materialized/sql.py

```diff
@@ -0,0 +1,27 @@
+FinalRiskOfBiasScore = (
```

Comment:
We need to separate the SQL statements that create/drop the materialized views from the migrations so that we can use them in commands. I thought it best to keep them in a separate module, and then import them into `models.py` for class specific create/drop, just so that `models.py` doesn't get clogged up with all of these lines. Do you like this approach, or should we just put this inline in `models.py`?
(I haven't replaced migrations with this yet, I'll do that after we find a home for these)